src/matter/session/secure/SecureChannelMessenger.ts

```diff
@@ -66,12 +66,19 @@ export class SecureChannelMessenger<ContextT> {
     }
 
     private async sendStatusReport(generalStatus: GeneralStatusCode, protocolStatus: ProtocolStatusCode) {
-        await this.exchange.send(MessageType.StatusReport, encodeStatusReport(generalStatus, protocolStatus));
+        const statusReportSchema = new SecureChannelStatusMessageSchema();
```

Comment:
You can create a constant for this schema in SecureChannelStatusMessageSchema.ts instead of creating a new instance each time, the same way it is done for all TlvSchema